### PR TITLE
Assetify crash font

### PR DESCRIPTION
**Pull request by nanaian**
Depends on https://github.com/decompals/pigment64/pull/16.

Have to do a cast because there's no way to ask splat to emit images as u32 arrays right now (although pigment64 has C array format support?).

### CODE REVIEW

File: ver/us/splat.yaml [hunk 1]
```diff
@@ -369,7 +369,12 @@ segments:
       - [0x511E0, vtx, theater/wall_shadows_vtx]
       - [0x51260]
       - [0x51540, ia8, ui/no_controller, 128, 32]
-    - [auto, .data, crash_screen]
+    - start: auto
+      type: .data
+      name: crash_screen
+      subsegments:
+      - [0x52690, i1, crash_screen/font, 32, 63]
```
Comment (ethteck) [inline ver/us/splat.yaml:376]: the start addr here can be auto I'd think

Comment (ethteck) [reply ver/us/splat.yaml:376]: nm

**Comment by ethteck** [comment]
this doesn't match unfortunately. the new rom is larger than the baserom, and it starts I think around the crash screen font area

File: ver/us/splat.yaml (hunk 1 above)
Comment (nanaian) [reply ver/us/splat.yaml:376]: This gives `Error: crash_screen/font should end at 0x4FADC, but it ends at 0x5278C`

**Pull request merged**